### PR TITLE
Send message to user/organization/community dialogs.

### PR DESCRIPTION
**Pull request by alstojanovic**
### Describe the background of your pull request

What does your pull request do? Does it solve a bug (which one?), add a feature?

### Additional context

Add any other context

### Governance 

- [ ] Documentation is added
- [ ] Test cases are added or updated

By submitting this pull request I confirm that:
- I've read the contributing document https://github.com/alkem-io/.github/blob/master/CONTRIBUTING.md.
- I've read and understand the Code of Conduct https://github.com/alkem-io/.github/blob/master/CODE_OF_CONDUCT.md.
- I understand that any contributions or suggestions I made may make it into the actual code. I've read the License 
     https://github.com/alkem-io/Coordination/blob/master/LICENSE.
- I understand that submitting the pull request requires agreeing to and signing the contributor license agreement.

### CODE REVIEW

File: src/common/components/composite/common/ContactLeadsButton/ContactLeadsButton.tsx [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+interface ContactLeadsButtonProps {
+  title?: string;
+  tooltip?: string;
+  sx?: IconButtonProps['sx'];
+  onClick: () => void;
+}
+
+const ContactLeadsButton: FC<ContactLeadsButtonProps> = ({ title = undefined, tooltip = '', sx, onClick }) => {
+  return (
+    <>
```
[the remark this answers is not in the document]
Comment (alstojanovic) [reply src/common/components/composite/common/ContactLeadsButton/ContactLeadsButton.tsx:14]: Thanks. It had something more, then I forgot to remove it...

File: src/domain/communication/messaging/DirectMessaging/DirectMessageDialog.tsx [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+  messageReceivers,
+  title,
+}) => {
+  const { t } = useTranslation();
+  const [isLoading, setIsloading] = useState(false);
+  const handleSendMessage = async (values: SendMessageData, { resetForm }) => {
+    setIsloading(true);
+    await onSendMessage(values.message);
+    resetForm();
+    setIsloading(false);
```
[the remark this answers is not in the document]
Comment (alstojanovic) [reply src/domain/communication/messaging/DirectMessaging/DirectMessageDialog.tsx:52]: Thanks

File: src/domain/community/contributor/user/layout/UserPageBanner.tsx [hunk 3]
```diff
@@ -10,10 +11,13 @@ import { useUserMetadata } from '../hooks/useUserMetadata';
 const UserPageBanner: FC = () => {
   const { userNameId = '' } = useUrlParams();
   const { user: currentUser } = useUserContext();
+  const handleError = useApolloErrorHandler();
```
[the remark this answers is not in the document]
Comment (alstojanovic) [reply src/domain/community/contributor/user/layout/UserPageBanner.tsx:14]: Remove from both places? We also have this in share dialog, remove from there as well?

**Review by ccanos — APPROVED** [review]

**Comment by ccanos** [comment]
approved with small comments. It works well in local
Resolving all the conversations and merge is alright

**Review by ccanos — APPROVED** [review]

**Pull request merged**